lib/tasks/hair_trigger.rake

```diff
@@ -11,6 +11,8 @@ namespace :db do
   namespace :schema do
     desc "Create a db/schema.rb file that can be portably used against any DB supported by AR"
     task :dump => :environment do
+      next unless ActiveRecord::Base.schema_format == :ruby
```

Comment:
This seems to break `db:migrate` in recent versions of Rails:
```
NoMethodError: undefined method `schema_format' for ActiveRecord::Base:Class (NoMethodError)
```

 It looks like `ActiveRecord::Base.schema_format` after [Rails v3.2.13 ](https://apidock.com/rails/v3.2.13/ActiveRecord/Base/schema_format/class) was deprecated and removed.  After that version it's just `ActiveRecord.schema_format` instead.

Reply:
good catch @jasonopslevel! fixed in #118 and released in v1.1.1 🤞